content/en/docs/concepts/experimental-features.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Experimental features"
+description: "Explanation of experimental features"
+type: docs
+weight: 10
+---
+
+Starting in version `v1.0.0-rc.4`, Notation began introducing experimental features. Experimental features are features that are not yet ready for production use but are available for testing and evaluation. Experimental features may change significantly in later releases or be completely removed. Experimental features are not enabled by default and are not supported.
+
+Currently, Notation has the following experimental features:
+
+- Signing, listing and verifying images as OCI image layout
```

Comment:
Maybe add a link for user guide, https://notaryproject.dev/docs/how-to/oci-image-layout/

Reply:
good call. Fixed